engine/class_modules/sc_warrior.cpp

```diff
@@ -245,6 +247,7 @@ struct warrior_t : public player_t
   {
     const spell_data_t* spell_reflection;
     const spell_data_t* recklessness;
+    const spell_data_t* bastion_of_might;
```

Comment:
```suggestion
```
You don't need to set spec.bastion_of_might for two reasons:
1) It doesn't belong in the spec. namespace because it's for spells associated with a spec's core components, as in spells that can be called with find_specialization_spell()
2) You're not using it anywhere